components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -2007,30 +2008,29 @@ private OMElement queryToGetAPIUsageByResourcePath(String columnFamily, String f
         Connection connection = null;
         Statement statement = null;
         ResultSet rs = null;
+        List<APIUsageByResourcePath> usage=new ArrayList<APIUsageByResourcePath>();
         try {
             connection = dataSource.getConnection();
             statement = connection.createStatement();
             String query;
 
             query = "SELECT api,version,apiPublisher,context,method,total_request_count,time FROM "
-                    + columnFamily + " WHERE " + APIUsageStatisticsClientConstants.TIME + " BETWEEN " +
+                    + tableName + " WHERE " + APIUsageStatisticsClientConstants.TIME + " BETWEEN " +
                     "\'" + fromDate + "\' AND \'" + toDate + "\'";
             rs = statement.executeQuery(query);
-            StringBuilder returnStringBuilder = new StringBuilder("<omElement><rows>");
-            int columnCount = rs.getMetaData().getColumnCount();
+            APIUsageByResourcePath apiUsageByResourcePath;
+
             while (rs.next()) {
-                returnStringBuilder.append("<row>");
-                for (int i = 1; i <= columnCount; i++) {
-                    String columnName = rs.getMetaData().getColumnName(i);
-                    String columnValue = rs.getString(columnName);
-                    returnStringBuilder.append("<" + columnName.toLowerCase() + ">" + columnValue +
-                            "</" + columnName.toLowerCase() + ">");
-                }
-                returnStringBuilder.append("</row>");
+                String apiName = rs.getString("api");
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621